datasets/br/bcb_disqualified_persons/bcb_disqualified_persons.yaml

```diff
@@ -0,0 +1,27 @@
+title: Brazil's Central Bank General Register of Disqualified Persons  
+entry_point: crawler.py
+prefix: bcb-disqualified-persons
+coverage:
+  frequency: daily
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  The QGI is a dataset from the Central Bank of Brazil listing individuals disqualified from certain roles in financial institutions and consortium administrators.
+description: |
+  The QGI, maintained by the Central Bank of Brazil,
+  serves as a crucial resource for monitoring regulatory compliance in the financial sector.
+  It enumerates individuals who, due to penalties incurred in administrative sanction proceedings,
+  are temporarily prohibited from occupying key positions such as administrators
+  or board members in financial or consortium administration entities.
+  This measure ensures the integrity and reliability of personnel in critical financial roles,
+  reflecting the Central Bank's commitment to maintaining stability and transparency in the financial system.
+  The dataset is a part of broader efforts to uphold high standards in financial governance
+  and protect the interests of stakeholders in the financial industry.
+publisher:
+  name: BCB (Brazil's Central Bank)
```

Comment:
I tend to put the official name in the local language in full in the name field - `Banco Central do Brasil (BCB)` - then I translate that in the publisher.description field.